src/abstractdataframe/io.jl

```diff
@@ -202,6 +202,60 @@ end
 
 ##############################################################################
 #
+# LaTeX output
+#
+##############################################################################
+
+function latex_char_escape(char::SubString)
+    if char == "\\"
+        return "\\textbackslash{}"
+    elseif char == "~"
+        return "\\textasciitilde{}"
+    else
+        return string("\\", char)
+    end
+end
+
+function latex_escape(cell::AbstractString)
+    cell = replace(cell, ['\\','~','#','$','%','&','_','^','{','}'], latex_char_escape)
+    return cell
+end
+
+function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame)
+    nrows = size(df, 1)
+    ncols = size(df, 2)
```

Comment:
Could be written more succinctly as `nrows, ncols = size(df)`. Doesn't matter too much, just thought I'd point it out.